lib/cxml/invoice_detail_request/accounting_segment.rb

```diff
@@ -0,0 +1,23 @@
+module CXML
+  module InvoiceDetailRequest
+    class AccountingSegment
+      attr_accessor :id, :name, :description
+
+      def initialize(data = {})
```

Comment:
This approach worries me a little, seems like it has a few red flags (passing around unlimited parameters, defensive coding etc) but I see it's a pattern that is already established in the gem you forked.